scripts/compare.ts

```diff
@@ -0,0 +1,401 @@
+  1843176, 1843177, 1843199, 1843201, 1843219, 1843247, 1843248, 1843344,
+  1843369, 1843393, 1843442, 1843448, 1843499, 1843536, 1843580, 1843585,
+  1843587, 1843592, 1843603, 1843608, 1843615, 1843616, 1843621, 1843627,
+  1843632, 1843633, 1843635, 1843640, 1843641, 1843644, 1843645, 1843646,
+  1843648, 1843651, 1843653, 1843654, 1843658, 1843660, 1843664, 1843665,
+  1843666, 1843676, 1843677, 1843693, 1843694, 1843696, 1843701, 1843708,
+  1843711, 1843720, 1843721, 1843725, 1843728, 1843732, 1843734, 1843742,
+  1843743, 1843746, 1843753, 1843760, 1843762, 1843764, 1843768, 1843769,
+  1843777, 1843779, 1843781, 1843791, 1843794, 1843795, 1843799, 1843800,
+  1843802, 1843803, 1843806, 1843809, 1843811, 1843812, 1843816, 1843817,
+  1843821, 1843822, 1843824, 1843825, 1843826, 1843829, 1843840, 1843842,
+  1843847, 1843856, 1843871, 1843873, 1843877, 1843882, 1843892, 1843893,
+  1843899, 1843905, 1843909, 1843913, 1843918, 1843919, 1843922, 1843923,
+  1843926, 1843932, 1843934, 1843935, 1843945, 1843950, 1843953, 1843954,
+  1843956, 1843960, 1843982, 1843993, 1843996, 1843999, 1844006, 1844015,
+  1844021, 1844023, 1844024, 1844026, 1844027, 1844052, 1844057, 1844061,
+  1844067, 1844075, 1844078, 1844083, 1844087, 1844091, 1844093, 1844094,
+  1844095, 1844096, 1844100, 1844102, 1844103, 1844106, 1844109, 1844112,
+  1844115, 1844117, 1844120, 1844129, 1844138, 1844141, 1844142, 1844144,
+  1844146, 1844149, 1844155, 1844158, 1844161, 1844162, 1844176, 1844184,
+  1844187, 1844188, 1844194, 1844199, 1844203, 1844208, 1844213, 1844220,
+  1844221, 1844233, 1844239, 1844240, 1844241, 1844247, 1844258, 1844266,
+  1844267, 1844276, 1844282, 1844284, 1844287, 1844294, 1844296, 1844324,
+  1844331, 1844341, 1844353, 1844364, 1844366, 1844380, 1844389, 1844391,
+  1844396, 1844399, 1844400, 1844403, 1844405, 1844413, 1844416, 1844417,
+  1844419, 1844421, 1844426, 1844432, 1844441, 1844453, 1844457, 1844494,
+  1844504, 1844510, 1844535, 1844580, 1844684, 1844692, 1844708, 1844768,
+  1844769, 1844774, 1844786, 1844798, 1844848, 1844853, 1844924, 1844953,
+  1844974, 1845096, 1845162, 1845217, 1845236, 1845297, 1845298, 1845328,
+  1845369, 1845416, 1845517, 1845535, 1845569, 1845578, 1845678, 1845679,
+  1845722, 1845731, 1845742, 1845790, 1845812, 1845814, 1845816, 1845832,
+  1845838, 1845890, 1845933, 1845934, 1845952, 1846013, 1846024, 1846095,
+  1846135, 1846209, 1846268, 1846383, 1846407, 1846421, 1846434, 1846440,
+  1846442, 1846485, 1846538, 1846568, 1846615, 1846626, 1846654, 1846723,
+  1846725, 1846736, 1846785, 1846792, 1846798, 1846828, 1846911, 1846916,
+  1846920, 1846954, 1847267, 1851770, 1853306, 1856560, 1857874, 1858843,
+  1862076, 1866430, 1867999, 1870625, 1871689, 1872721, 1875733, 1876039,
+  1877112, 1877994, 1878136, 1880942, 1887033, 1891663, 1891667, 1891681,
+  1891703, 1891779, 1891991, 1892061, 1892111, 1892151, 1892152, 1892181,
+  1892563, 1893416, 1894739, 1895907, 1898781, 1900291,
+]);
```

Comment:
For the mainnet we don't need this kind of work? How we can make this code mainnet ready?

Reply:
no, they are wrong blocks in the harvester